src/scss/_components.respondents.scss

```diff
@@ -0,0 +1,364 @@
+
+          @media (min-width: $bp-respond-medium-layout) {
+            --size: 12rem;
+          }
+
+          @media (min-width: $bp-respond-size-bump-medium) {
+            --size: 14rem;
+          }
+      
+          @media (min-width: $bp-respond-size-bump-large) {
+            --size: 16rem;
+          }
+        }
+      }
+
+      &--managers {
+        top: 1rem;
+
+        @media (min-width: $bp-respond-stop-overlap) {
+          top: -1rem;
+          left: calc(100% - 18rem);
+        }
+
+        @media (min-width: $bp-respond-medium-layout) {
+          top: 0.125rem;
+          left: calc(100% - 13rem);
+        }
+
+        @media (min-width: $bp-respond-size-bump-medium) {
+          top: -0.5rem;
+          left: calc(100% - 14.5rem);
+        }
+
+        @media (min-width: $bp-respond-size-bump-large) {
+          left: calc(100% - 19rem);
+        }
+
+        .cmp-diamond{
+          --size: 7rem;
+
+          @media (min-width: $bp-respond-stop-overlap) {
+            --size: 6rem;
+          }
+
+          @media (min-width: $bp-respond-medium-layout) {
+            --size: 3.5rem;
+          }
+
+          @media (min-width: $bp-respond-size-bump-medium) {
+            --size: 5rem;
+          }
+
+          @media (min-width: $bp-respond-size-bump-large) {
+            --size: 7rem;
+          }
+        }
+      }
+
+      &--other {
+        @media (min-width: $bp-respond-stop-overlap) {
+          top: 1rem;
+          left: calc(100% - 15rem);
+        }
+
+        @media (min-width: $bp-respond-medium-layout) {
+          top: 1.5rem;
+          left: calc(100% - 9rem);
+        }
+
+        @media (min-width: $bp-respond-size-bump-medium) {
+          top: 2rem;
+          left: calc(100% - 10.5rem);
+        }
+
+        @media (min-width: $bp-respond-size-bump-large) {
+          left: calc(100% - 14.5rem);
+        }
+
+        .cmp-dodecahedron{
+          --size: 5rem;
+
+          @media (min-width: $bp-respond-medium-layout) {
+            --size: 2.25rem;
+          }
+
+          @media (min-width: $bp-respond-size-bump-medium) {
+            --size: 2.75rem;
+          }
+
+          @media (min-width: $bp-respond-size-bump-large) {
+            --size: 4rem;
+          }
+        }
+      }
+    }
+  }
+
+  &__stat {
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    position: relative;
+    z-index: 100;
+    text-align: center;
+    text-shadow: 0 0 1rem $black, 0 0 2rem $black, 0 0 1rem $black, 0 0 4rem $black;
+
+    @media (min-width: $bp-respond-stop-overlap) {
+      text-align: right;
+      justify-content: flex-end;
+    }
+  }
+
+  &__group {
+    font-size: 1.5rem;
+    font-weight: bold;
+    line-height: 1.4;
+
+    @media (min-width: $bp-respond-size-bump-large) {
+      font-size: 1.875rem;
+    }
+
+    &--designers {
+      color: $hotpink;
+
+      @supports (display: grid) {
+        color: mix($white, $hotpink, 80%);
```

Comment:
This is subjective feedback. The words before we get to the `$bp-respond-stop-overlap` breakpoint are a bit difficult to read, this pink one especially. I bumped the percentage up to 85% locally, and that helped some. Any higher than that percentage and they may as well just be white, which seems like it could be an option too. Just "typing out loud" here, feel free to take or leave this 👍